Objective-C/CBLDatabase.mm

```diff
@@ -302,14 +312,14 @@ - (BOOL) delete: (NSError**)outError {
         [self mustBeOpen];
         
         if (_activeReplications.count > 0) {
-            NSString* err = @"Cannot delete the database. Please stop all of "
-            "the replicators before closing the database.";
+            NSString* err = @"Cannot delete the database. "
+                "Please stop all of the replicators before closing the database.";
```

Comment:
this is delete() method. message is not correct

Reply:
Thanks for catching this.